### PR TITLE
Add patient data and deletion- First commit

### PR DESCRIPTION
**Pull request by AieshaShetty**
# Pull Requests Requirements

- [ ] The PR title includes a brief description of the work done, including the Issue number if applicable.
- [ ] The PR includes a video showing the changes for the work done.
- [ ] The PR title follows conventional commit label standards.
- [ ] The changes confirm to the OpenElis Global x3 Styleguide and design documentation.
- [ ] The changes include tests or are validated by existing tests.
- [ ] I have read and agree to the Contributing Guidelines of this project.

## Screenshots

[Add relevant screenshots here if applicable]

## Related Issue

[Add a link to the related issue or mention it here if applicable]

## Other

[Add any additional information or notes here]

### CODE REVIEW

**Review by njiddasalifu — COMMENTED** [review]
@AieshaShetty could add abut summary of the work and also the ticket  link you are addressing.

File: src/test/java/org/openelisglobal/patient/PatientServiceTest.java [hunk 1]
```diff
@@ -25,7 +25,7 @@
 @ContextConfiguration(classes = { BaseTestConfig.class, PatientTestConfig.class })
 @TestPropertySource("classpath:common.properties")
 @ActiveProfiles("test")
-public class PatientServiceTest {
```
Comment (sherrif10) [inline src/test/java/org/openelisglobal/patient/PatientServiceTest.java:28]: Why this change ?

Comment (AieshaShetty) [reply src/test/java/org/openelisglobal/patient/PatientServiceTest.java:28]: Hi, I am honestly sorry for the late reply. Here, I want to check if I was able to integrate any changes to this application or not. 
Integrating bearer token authentication into the patient's service test enhances security, ensures compliance, and protects patient data. It's a significant contribution that strengthens access control and aligns with industry standards, benefiting the team's expertise and future projects. It's the initial stage of my contribution. It will take a couple of days to figure out how to make any further changes. My first milestone will be to modify the OpenELIS application code to interact with the search engine and implement functionality to index data into the search engine when records are created, updated, or deleted in the OpenELIS database.

**Comment by AieshaShetty** [comment]
@njiddasalifu Hi, I am honestly sorry for the late reply. Here, I want to check if I was able to integrate any changes to this application or not.
Integrating bearer token authentication into the patient's service test enhances security, ensures compliance, and protects patient data. It's a significant contribution that strengthens access control and aligns with industry standards, benefiting the team's expertise and future projects. It's the initial stage of my contribution. It will take a couple of days to figure out how to make any further changes. My first milestone will be to modify the OpenELIS application code to interact with the search engine and implement functionality to index data into the search engine when records are created, updated, or deleted in the OpenELIS database.

**Comment by njiddasalifu** [comment]
> @njiddasalifu Hi, I am honestly sorry for the late reply. Here, I want to check if I was able to integrate any changes to this application or not. Integrating bearer token authentication into the patient's service test enhances security, ensures compliance, and protects patient data. It's a significant contribution that strengthens access control and aligns with industry standards, benefiting the team's expertise and future projects. It's the initial stage of my contribution. It will take a couple of days to figure out how to make any further changes. My first milestone will be to modify the OpenELIS application code to interact with the search engine and implement functionality to index data into the search engine when records are created, updated, or deleted in the OpenELIS database.

Description clear. @sherrif10 can give more reviews
